nilearn/plotting/html_stat_map.py

```diff
@@ -71,6 +72,15 @@ def _threshold_data(data, threshold=None):
         data = np.ma.masked_equal(data, 0, copy=False)
     else:
         data = np.ma.masked_inside(data, -threshold, threshold, copy=False)
+
+    # if all values above threshold, np.ma returns np.bool_(False) ;
+    # this will fail our shape check, so we'll convert to an array
+    if not np.ndim(data.mask):
+        value_check = data.min()
+        warnings.warn("The given float value must not be less than {0}. "
```

Comment:
Maybe I would phrase this as "Threshold given was ..., but the data has not values above ...".

Reply:
Done, thanks ! :sparkles:

Reply:
> Maybe I would phrase this as "Threshold given was ..., but the data has not values above ...".

shouldn't it be "below" rather than "above"?